### PR TITLE
Suggest name for public key too

### PR DESCRIPTION
**Pull request by rikhuijzer**
For consistency with the suggestion for the private key name in the code here and for consistency with many tutorials which advice setting `DOCUMENTER_PUB`.

### CODE REVIEW

**Comment by mortenpi** [comment]
Actually, it should not be necessary to give it a title at all. We set `-C Documenter`, so you should get something like

```
ssh-rsa AAAAB...z714rOGBK10= Documenter
```

and, if I remember correctly, GitHub uses that key comment as the name, if the "Title" field is left empty.

**Comment by rikhuijzer** [comment]
> And, if I remember correctly, GitHub uses that key comment as the name if the "Title" field is left empty.

I had no idea about that. Maybe we can incorporate that in the text without being too verbose? Otherwise, feel free to close this PR.

**Comment by mortenpi** [comment]
> Maybe we can incorporate that in the text without being too verbose?

Yeah, that's a good idea. Happy to take suggestions on how to rephrase it :slightly_smiling_face:

**Comment by rikhuijzer** [comment]
> And, if I remember correctly, GitHub uses that key comment as the name if the "Title" field is left empty.

I just tested it and it indeed it works.

I've rephrased the sentence and put the sentence in the order of the fields in the GitHub interface.

File: src/genkeys.jl [hunk 1]
```diff
@@ -62,7 +62,8 @@ function genkeys(; user="\$USER", repo="\$REPO")
 
     # Prompt user to add public key to github then remove the public key.
     let url = "https://github.com/$user/$repo/settings/keys"
-        @info("add the public key below to $url with read/write access:")
+        @info("add a public key to $url with an empty title (GitHub will infer it), " *
+              "the following key, and read/write access.")
```
Comment (mortenpi) [inline src/genkeys.jl:66]:
```suggestion
        @info """
        add the public key below to $url with read & write access
        the title can be left empty, as GitHub can infer it from the key comment
        """
```

**Review by mortenpi — APPROVED** [review]

**Comment by mortenpi** [comment]
Thanks @rikhuijzer!

**Pull request merged**